### PR TITLE
Updated Heartcore MVC Sample

### PR DESCRIPTION
**Pull request by devantler**
Updated Heartcore MVC Sample to reflect the changes in <https://github.com/umbraco/Umbraco.Headless.Client.Net/pull/50#issue-1302122023>

### CODE REVIEW

File: Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/MVC-Sample/index.md [hunk 1]
```diff
@@ -109,24 +121,27 @@ namespace Umbraco.Headless.Client.Samples.Web.Controllers
...
+The controller is now in place, but to show our content, we also need to define a view.
 
-6. Create a folder in `/Views` using the alias of the Document Type, e.g. `/HomePage`
-7. Create an `Index.cshtml` file in the new folder
-8. Follow steps 2-3 from the ['Define a view file' section](#define-a-view-file).
-9. Build and run the solution
+1. Create a folder in `/Views` using the alias of the Document Type, e.g. `/HomePage`
+2. Create an `Index.cshtml` file in the new folder
+3. Follow steps 2-3 from the ['Define a view file' section](#define-a-view-file).
+4. Build and run the solution
```
Comment (eshanrnh) [inline Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/MVC-Sample/index.md:129]:
```suggestion
1. Create a folder in `/Views` using the alias of the Document Type, e.g. `/HomePage`.
2. Create an `Index.cshtml` file in the new folder.
3. Follow steps 2-3 from the [Define a view file](#define-a-view-file) section.
4. Build and run the solution.
```

File: Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/MVC-Sample/index.md [hunk 2]
```diff
@@ -109,24 +121,27 @@ namespace Umbraco.Headless.Client.Samples.Web.Controllers
...
 
 ## Building view files
 
-In order to render out the data from the properties on our content we need to use the `@Model.Value("")` approach, where the value will be the alias of the property you want to display data from. 
+In order to render out the data from the properties on our content, we need to use the `@Model.PropertyName` approach, where the value value of the property you want to display data from.
```
Comment (eshanrnh) [inline Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/MVC-Sample/index.md:133]:
```suggestion
To render out the data from the properties on our content, we need to use the `@Model.PropertyName` approach, where the value is the Name of the property you want to display the data from.
```

File: Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/MVC-Sample/index.md [hunk 3]
```diff
@@ -109,24 +121,27 @@ namespace Umbraco.Headless.Client.Samples.Web.Controllers
...
 
-An example could be a textstring property with the alias `heading`. To render the data from this property on the frontend, we will need to use `@Model.Value("heading")`.
+An example could be a text string property with the alias `heading`. To render the data from this property on the frontend, we will need to use `@Model.Heading`.
 
-Below is a full example of how a basic view for a root node could look.
+> **Note**
+> To render data from a property the property must be defined in the view model (`@model`), and it must match an alias on the corresponding content node from your Umbraco project.
```
Comment (eshanrnh) [inline Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/MVC-Sample/index.md:138]:
```suggestion
:::note
To render data from a property, the property must be defined in the view model (`@model`), and it must match an alias on the corresponding content node from your Umbraco project.
:::
```

**Review by eshanrnh — COMMENTED** [review]
Looks good, @devantler 🙌 I've added a couple of suggestions.

By when do you want this PR to be merged?

File: Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/MVC-Sample/index.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
-In `/samples/Umbraco.Headless.Client.Samples.Web/` you will find a .NET Core 2.2 based MVC website implementation. It presents one possible approach to creating a website using Umbraco Heartcore for Content Delivery.
+In `/samples/Umbraco.Headless.Client.Samples.Web/` you will find a .NET 6.0-based MVC website implementation. It presents one possible approach to creating a website using Umbraco Heartcore for Content Delivery.
 
-The sample is built up around a sample project with the alias `demo-headless`. You can choose to test the sample with the sample project or connect the sample to your own project.
+The sample is built around a sample project with the alias `demo-headless`. You can choose to test the sample with the sample project or connect the sample to your own project.
 
 ## Prerequisites
 
-- [.NET Core SDK 2.2](https://dotnet.microsoft.com/download/dotnet-core/2.2)
+- [.NET 6.0](https://dotnet.microsoft.com/download/dotnet-core/2.2)
```
[the remark this answers is not in the document]
Comment (devantler) [reply Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/MVC-Sample/index.md:9]: Awesome, I will have a look at those ASAP. It should be merged when [this PR](https://github.com/umbraco/Umbraco.Headless.Client.Net/pull/50#event-6983694262) is merged. In this way, it matches the code :-)

**Comment by eshanrnh** [comment]
Ah, ok 😀 We have a standard look for Notes as mentioned in the [Markdown conventions](https://our.umbraco.com/Documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips) article which display Notes as below: 
image

**Comment by rasmusjp** [comment]
FYI: I've merged the code, so this can be merged now 🚀

**Pull request merged**